front-end/src/components/LeafletMap.vue

```diff
@@ -49,26 +49,43 @@ let leaflet: L.Map | undefined;
 
 const markerTable = new Map<string, Marker<null>>();
 
+const DEFAULT_LAT = 41.87213786;
+const DEFAULT_LONG = -87.62576558;
+const DEFAULT_ZOOM_OUT = 14;
+const DEFAULT_ZOOM_IN = 18;
+
+const isProd = import.meta.env.PROD;
+
+const defaultIcon = L.icon({
+  iconUrl: isProd
+    ? 'https://arches-app-demo.opentechstrategies.com/archesdataviewer/marker-icon.png'
```

Comment:
Is there any way we can abstract out `https://arches-app-demo.opentechstrategies.com/archesdataviewer/` to a constant like `FRONT_END_ROOT` or something?

Reply:
Now that you mention it, there already is a variable! `import.meta.env.VITE_ARCHES_API_URL`. Agreed that it would be cleaner that way, allow me to make that change